mono/mini/debugger-agent.c

```diff
@@ -6016,7 +6526,11 @@ mono_debugger_agent_handle_exception (MonoException *exc, MonoContext *throw_ctx
 		tls->catch_state.valid = TRUE;
 	}
 
+#ifndef IL2CPP_MONO_DEBUGGER
 	process_event (EVENT_KIND_EXCEPTION, &ei, 0, throw_ctx, events, suspend_policy);
+#else
+	process_event (EVENT_KIND_EXCEPTION, &ei, 0, throw_ctx, events, suspend_policy, 0);
```

Comment:
Again, it feels like we could always pass NULL in both runtimes in most calls to process_event and get rid of a ton of these #defines...also pass in NULL on mono runtime when il2cpp passes something else